orm/src/main/java/org/hibernate/search/batchindexing/impl/IdentifierConsumerDocumentProducer.java

```diff
@@ -24,6 +24,7 @@
 import org.hibernate.engine.spi.SessionFactoryImplementor;
 import org.hibernate.engine.spi.SessionImplementor;
 import org.hibernate.engine.transaction.jta.platform.spi.JtaPlatform;
+import org.hibernate.internal.CriteriaImpl;
```

Comment:
Are we absolutely sure that this class won't change or (more importantly) get repackaged somewhere else before ORM 6.0? Just asking because of what happened in ORM 5.2...

It would be a shame that some previous version of Search ends up non-compatible with a new (possibly micro) version of ORM just because we wanted to get rid of a warning...